package.json

```diff
@@ -1,19 +1,21 @@
 {
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap && lerna link --force-local",
     "docs": "x0 docs",
     "docs:build": "x0 build docs",
     "docs:deploy": "npm run docs:build && now dist --public && now alias",
     "test": "lerna run test",
-    "format": "prettier --no-semi --single-quote --write '{examples,packages}/**/*.js'",
-    "prepare": "lerna bootstrap && lerna link --force-local"
```

Comment:
I removed the `prepare` script because `lerna bootstrap` in Lerna v3 triggers that script, so it causes an endless loop. It would be ideal if we could keep this, because then `lerna bootstrap` would happen automatically and contributors and Travis CI wouldn't have to know about it, so I'm open to ideas.